parser_email.py

```diff
@@ -786,7 +786,23 @@ def _add_body_in_email_headers(parsed_mail, file_path, charset, content_type):
             body_content = f.read()
         _debug_print("Reading file data using binary mode")
     # Add body to the last added Email artifact
-    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8')
+    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8').replace('\u0000', '')
```

Comment:
The additional `replace` is to remove NULL character, which should improve the final result.